mim_ocr/graph/builders.py

```diff
@@ -0,0 +1,180 @@
+from typing import Tuple
+from abc import ABC, abstractmethod
+from enum import Enum
+
+import numpy as np
+from igraph import Vertex, Graph, Edge
```

Comment:
Czy `igraph` jest w zależnościach? Nigdzie go nie znalazłam.

Reply:
Dorzuciłem do zwykłych requirements.txt ( z tego co widzę to tych requirementsów jest kilka conajmniej, nie wiem które powinno być odpowiednie szczerze)

Reply:
Zgadzam się, że nawigacja po requirementsach jest niejasna. Dodałam krótką dokumentację w #18 – proszę przejrzyj, czy ta instrukcja jest wystarczająca.

Zwykłe requirements.txt to dobre miejsce. Proszę dodaj jeszcze w `pyproject.toml`.

Reply:
Jasne, dodałem.